### DISCUSSION TITLE
Create new collection

### DISCUSSION THREAD

**Discussion by Oggy4th** [pocketbase/pocketbase#2427] [Q&A]
What is difference between create new collection with data validations and without data validations?

**Comment by ganigeorgiev** [comment]
`Dao.SaveCollection()` is roughly equivalent to a direct INSERT/UPDATE statement in the database. We perform some minimal constraints checks to ensure that the database is not left in a broken state but allow users to set practically any name, id, etc. they want. This method should be used as part of a migration or when working with trusted/verified input.

The [`forms` package](https://pkg.go.dev/github.com/pocketbase/pocketbase/forms) structs on the other hand are safe to be populated with untrusted/unverified input and are intended to be used when working with external data (eg. data coming from an API request). Internally they use the `Dao.*` methods.
